static/js/table.js

```diff
@@ -170,7 +170,7 @@ function renderTable (id, options = {}, renderRow) {
     for (let i = 0; i < data.length; i++) {
       const item = data[i]
       const foundIndex = findIndex(t.rows, start, item)
-      if (foundIndex !== -1) {
+      if (foundIndex !== -1 && item.error === undefined) {
```

Comment:
you want to check for the error property earlier, not when we've already tried to "find" it.